services/web/src/common/components/SVGIcon/SVGIcon.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SemanticElement } from 'common/helpers';
+import Icons from './Icons';
+import { COLORS, SIZES } from './const';
+
+import './svg-icon.less';
+
+export default class SVGIcon extends SemanticElement {
```

Comment:
why call it a `SemanticElement` is there a connection with semantic-ui ? 
wondering if bedrockElement would be better ?

Reply:
Long story short `SVGIcon` is meant as a replacement for semantic `Icon` which uses font icons but I wanted the API to match (mostly so that Dom wouldn't bitch at me when using it) ... also I still kind of like Semantic's style... specifically adding very readable semantic classes like `ui menu item` etc and wanted to follow this. So basically `SemanticElement` has nothing to do with Semantic directly but more inspired by it... open to renaming or other ideas here.

Reply:
wait I didn't see that last line... `extends BedrockElement`?? ooo I like that!

Reply:
Abandoning this one entirely. I have some ideas for how to better tame semantic, but I'll compile them and make a new PR.